php/libraries/NDB_Menu.class.inc

```diff
@@ -142,6 +142,9 @@ class NDB_Menu extends NDB_Page
      */
     function display()
     {
+        if ($this->skiptemplate) {
+		return "";
```

Comment:
the return type of "display()" is a string to insert into the template. (The other code paths all return a string.)

I updated the doc comment to be more accurate, since it clearly isn't returning "void"..